aries-site/src/examples/templates/wizard/StickyHeaderFooterExample.js

```diff
@@ -0,0 +1,286 @@
+    ),
+  },
+  {
+    description: 'Step 2 description.',
+    inputs: (active, setActive) => (
+      <StepTwo active={active} setActive={setActive} />
+    ),
+  },
+  {
+    description: 'Provide a summary of what was accomplished or configured. ',
+    inputs: (active, setActive) => (
+      <StepThree active={active} setActive={setActive} />
+    ),
+    title: 'Finish title',
+  },
+];
+
+export const StickyHeaderFooterExample = () => {
+  const [active, setActive] = useState(1);
+
+  // store form values in state so they persist
+  // when user goes back a step
+  const [formValues, setFormValues] = useState({});
+
+  // controls state of cancel layer
+  const [open, setOpen] = useState(false);
+  const size = useContext(ResponsiveContext);
+  return (
+    <>
+      <Box fill>
+        <Header
+          border={{ side: 'bottom', color: 'border-weak' }}
+          pad="small"
+          fill="horizontal"
+          justify="center"
+          flex={false}
+        >
+          <Box direction="row" flex>
+            {active > 1 && (
+              <Button
+                label={size !== 'small' ? `Step ${active - 1}` : undefined}
+                icon={<FormPreviousLink color="text-strong" />}
+                onClick={() => setActive(active - 1)}
+              />
+            )}
+          </Box>
+          <Box>
+            <Text color="text-strong" weight="bold">
+              Action Title
+            </Text>
+          </Box>
+          <Box direction="row" flex justify="end">
+            <Button
+              label={size !== 'small' ? 'Cancel' : undefined}
+              icon={<FormClose color="text-strong" />}
+              reverse
+              onClick={() => setOpen(true)}
+            />
+          </Box>
+        </Header>
+        <Box
+          align="center"
+          pad={size !== 'small' ? 'large' : 'medium'}
+          //   flex={false}
+          flex
+          overflow="auto"
+        >
+          <Box width="medium" gap="medium">
+            <Box gap="medium" flex={false}>
+              <Box>
+                <Heading color="text-strong" margin="none">
+                  {steps[active - 1].title || `Step ${active} Title`}
+                </Heading>
+                <Text
+                  color="text-strong"
+                  size="small"
+                >{`Step ${active} of ${steps.length}`}</Text>
+              </Box>
+              <Text color="text-strong" size="large">
+                {/* Index an array starts at 0 */}
+                {steps[active - 1].description}
+              </Text>
+            </Box>
+            <Form
+              value={formValues}
+              onChange={setFormValues}
+              onSubmit={({ value }) => console.log(value)}
+            >
+              {/* Index an array starts at 0 */}
+              {steps[active - 1].inputs(active, setActive)}
+            </Form>
+          </Box>
+        </Box>
+        <Footer
```

Comment:
This is feeling like the button is too far removed from form steps. Is this trying to avoid having the button "jump" around from step to step?

 
![Screen Shot 2020-07-31 at 2 06 00 PM](https://user-images.githubusercontent.com/1756948/89073499-dc3f0100-d337-11ea-90c3-d8cd670ade6c.png)

Reply:
This to demonstrate where the next/finish button persists in the across all steps in the same position. I agree it does feel far away from the steps though